uniswap-v2/src/modules/11_store_cumulative_fields.rs

```diff
@@ -0,0 +1,40 @@
+use substreams::store::{DeltaBigDecimal, Deltas, StoreAdd};
+use substreams::store::{StoreAddBigDecimal, StoreNew};
+
+use crate::store_key::StoreKey;
+use crate::utils;
+
+#[substreams::handlers::store]
+pub fn store_cumulative_fields(
+    volume_deltas: Deltas<DeltaBigDecimal>,
+    output_store: StoreAddBigDecimal,
+) {
+    for delta in volume_deltas.deltas {
+        if let Some(pool_address) = StoreKey::Volume.get_pool(&delta.key) {
+            let volume = utils::delta_value(&delta);
+            let (supply_side_revenue, protocol_side_revenue) =
+                utils::calculate_revenue(volume.clone());
+
+            output_store.add(
```

Comment:
Putting this line before `let (supply_side_revenue, protocol_side_revenue) = utils::calculate_revenue(volume.clone());` will allow you to avoid cloning volume variable